controllers/model/modelversion_controller.go

```diff
@@ -139,6 +139,11 @@ func (r *ModelVersionReconciler) Reconcile(req ctrl.Request) (ctrl.Result, error
 			modelVersionStatus.Image = modelVersion.Spec.ImageRepo + ":v" + versionId
 			modelVersionStatus.ImageBuildPhase = modelv1alpha1.ImageBuilding
 			modelVersionStatus.Message = "Image building started."
+			imgBuildDockerfile := createImageBuildDockerfile(modelVersion.Namespace)
```

Comment:
The configMap is the same for all. But invocation here could cause creation many times